public/app/types/accessControl.ts

```diff
@@ -38,4 +44,8 @@ export enum AccessControlAction {
 export enum AccessControlScope {
   UsersAll = 'users:*',
   UsersSelf = 'users:self',
+
+  OrgAllUsersAll = 'org:*/users:*',
+  OrgCurrentUsersAll = 'org:current/users:*',
+  OrgCurrentUsersSelf = 'org:current/users:self',
 }
```

Comment:
I don't think the scopes should be sent to the frontend, they just don't matter there.